config/filesystems.php

```diff
@@ -55,33 +55,18 @@
             'visibility' => 'public',
         ],
 
-//        's3' => [
-//            'driver' => 's3',
-//            'key' => env('AWS_ACCESS_KEY_ID'),
-//            'secret' => env('AWS_SECRET_ACCESS_KEY'),
-//            'region' => env('AWS_DEFAULT_REGION'),
-//            'bucket' => env('AWS_BUCKET'),
-//            'url' => env('AWS_URL'),
-//        ],
-
-        // https://github.com/Superbalist/laravel-google-cloud-storage
-        'gcs-public-static' => [
-            'driver' => 'gcs',
-            'project_id' => env('GOOGLE_CLOUD_PROJECT_ID', 'your-project-id'),
-            'key_file' => env('GOOGLE_CLOUD_STORAGE_KEY_FILE', null), // optional: /path/to/service-account.json
-            'bucket' => env('GOOGLE_CLOUD_STORAGE_BUCKET', 'your-bucket'),
-            'path_prefix' => env('GOOGLE_CLOUD_STORAGE_PATH_PREFIX', null), // optional: /default/path/to/apply/in/bucket
-            'storage_api_uri' => env('GOOGLE_CLOUD_STORAGE_API_URI', null), // see: Public URLs below
-            'visibility' => 'public', // optional: public|private
+        's3-static' => [
+            'driver' => 's3',
+            'key' => env('STATIC_STORAGE_ACCESS_KEY'),
+            'secret' => env('STATIC_STORAGE_SECRET_KEY'),
+            'region' => env('STATIC_STORAGE_REGION', 'us-east-1'),
+            'bucket' => env('STATIC_STORAGE_BUCKET_NAME'),
+            'url' => env('STATIC_STORAGE_URL'),
+            'endpoint' => env('STATIC_STORAGE_ENDPOINT'),
+            'bucket_endpoint' => boolval(env('STATIC_STORAGE_USE_BUCKET_ENDPOINT', '0')),
```

Comment:
I deliberately went for USE in the env var name as otherwise it sounds a lot like it wants to be passed an URL. Like this the odd naming choice in the driver config is hidden from us.

Reply:
If there's a nicer way to get a boolean from a string in PHP please lmk.

Reply:
I think the naming is useful and also I think that is a good way "to bool" :+1: